pyinteraph/path_analysis.py

```diff
@@ -290,7 +292,28 @@ def filter_graph(graph, node_threshold, edge_threshold):
         log.warning("No metapaths found.")
     return filterd_graph
 
-def get_metapath(graph, res_id, res_space, node_threshold, edge_threshold):
+def normalize_graph(graph, normalize):
+    """Takes in a graph where all weights ranged between 0 to 1 and 
+    returns a graph where all weights are normalized (divided by max
+    weight)
+    """
+    normalized_graph = nx.Graph()
```

Comment:
Leave a blank line between the function description and the beginning of the code block.